climakitae/explore/vulnerability.py

```diff
@@ -244,10 +244,13 @@ def _check_params_get_name(
     if approach == "warming_level" and warming_level not in [
         "1.5",
         "2.0",
+        "2.5",
         "3.0",
         "4.0",
     ]:
-        raise ValueError("Passed in warming level must be '1.5', '2.0', '3.0', '4.0'")
+        raise ValueError(
+            "Passed in warming level must be '1.5', '2.0', '2.5', '3.0', '4.0'"
```

Comment:
Noting that the filename exported for vulnerability data at 2.5 warming level is "... _2degreeWL_..." so we need to modify the filename to indicate the 2.5 piece. Wondering here now if we do "2pt5WL" for the 1.5 and 2.5. What do you think @claalmve ?

Reply:
Makes sense to me!